### DISCUSSION TITLE
Some magic mod "Break"-related spells not dropping items

### DISCUSSION THREAD

**Discussion by ghost** [IzzelAliz/Arclight#1360] [General]
I tried to use Arclight on a 1.20.1 Forge (Arclight trails 1.0.5) SMP server with my friends, and installed Ars Nouveau + Mana and Artifice(MnA) on it.
As for plugins, I installed the Bukkit version of Luckperms and EssencialsX.

It mostly worked, but the "Fell"(chopping down trees) spell of Ars and the "Break" spell of MnA isn't dropping the blocks they broke.
On the other side, the "Break" spell of Ars works correctly.

Is this an issue, or do I have to change the config of Arclight and/or the plugins?
I know next to nothing about coding, so forgive my lack of knowledge.

**Comment by Mexerex12** [comment]
did u solve that?
